docs/assets/css/app.scss

```diff
@@ -45,3 +46,53 @@ pre {
     word-break: break-word;
   }
 }
+
+// Header overrides
+// Fixes service name placement and style
+.app-header {
```

Comment:
Should we split this code out into its own file?

Reply:
Done @edwardscull